dev.py

```diff
@@ -485,6 +483,12 @@ def setup_build(cls, dirs, args):
             cmd += ['-Db_sanitize=address,undefined']
         if args.setup_args:
             cmd += [str(arg) for arg in args.setup_args]
+        if args.use_scipy_openblas:
+            cls.configure_scipy_openblas()
+            env['PKG_CONFIG_PATH'] = os.pathsep.join([
```

Comment:
Possibly use `pathlib.Path` instead of `os.path` for future path manipulations. They tend to look a lot cleaner.

Reply:
I kept this in sync with the numpy version (see https://github.com/numpy/numpy/pull/24839) as much as possible. That seemed more important to me than `pathlib` vs. `os.path`. Mind if I keep it like this for now? We may anyway remove `dev.py` at some point, if `spin` ever catches up in functionality.

Reply:
Fine to keep for now, but I think pathlib would be much cleaner for future work

Reply:
I have to say I don't have a preference yet. `os.path` is very robust, while for example `Path.abspath()` is known to have a bug on Windows until Python 3.10, so it's unusable there and you must use `.resolve()` instead.